test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -25,12 +25,13 @@ notifyFormat_ngsi10
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 0 IPv4 -mutexTimeStat
+brokerStart CB 0 IPv4 -mutexTimeStat -subCacheIval 0
```

Comment:
There was a bug related with this due to the subscriptoin cache that has been now fixed (https://github.com/telefonicaid/fiware-orion/issues/1186). Maybe this was related and it is no longer needed as the bug has been solved?